LEGACY_SSE_EMULATION.md

```diff
@@ -23,21 +29,30 @@ emulate_legacy_sse_apis = ["V1"]
 (...)
 ```
 
-This will expose three additional sse endpoints:
+This setting will expose three legacy SSE endpoints with the following events streamed on each endpoint:
+
+- `/events/main` - `ApiVersion`, `BlockAdded`, `DeployProcessed`, `DeployExpired`, `Fault` and `Shutdown`
+- `/events/deploys`- `ApiVersion`, `DeployAccepted` and `Shutdown`
+- `/events/sigs` - `ApiVersion`, `FinalitySignature` and `Shutdown`
+
+Those endpoints will emit events in the same format as the legacy SSE API of the Casper node.
+
+## Event Mapping
 
-- `/events/sigs` -> publishes `ApiVersion`, `BlockAdded`, `DeployProcessed`, `DeployExpired`, `Fault` and `Shutdown`
-- `/events/deploys`-> publishes `ApiVersion`, `TransactionAccepted` and `Shutdown`
-- `/events/main` -> publishes `ApiVersion`, `FinalitySignature` and `Shutdown` events
+There are limitations to what the Casper Sidecar can and will do. Below, you will find a list of mapping assumptions between 2.x events and 1.x events.
 
-Those endpoints will emit events in the same format as the V1 SSE API of the casper node. There are limitations to what Casper Sidecar can and will do, here is a list of mapping assumptions:
+### The `ApiVersion` event
 
-## Translating `ApiVersion` event
+The legacy SSE ApiVersion event is the same as the current version.
 
-Legacy SSE event will be the same
+### The `BlockAdded` event
 
-## Translating `BlockAdded` event
+The Sidecar can emit a legacy `BlockAdded` event by unwrapping the 2.x event structure and creating a 1.x emulated event structure. 
 
-- When the 2.x event emits a V1 block it will be unwrapped and passed as a legacy BlockAdded, for instance a 2.x event like this:
+A Version1 `BlockAdded` event will be unwrapped and passed as a legacy `BlockAdded` event as shown below.
+
+<details>
```

Comment:
HTML in markdown is not a default feature. I would advise not to use it. If the file is to big we can split into smaller files.

Reply:
In this file, it's possible to remove the HTML. I think the length is reasonable, and I've added some links.

<img width="481" alt="Screenshot 2024-06-07 at 14 11 29" src="https://github.com/casper-network/casper-sidecar/assets/4185994/360ae8db-ecca-43a2-8c4b-606738fe6fe0">